requirements-hpu.txt

```diff
@@ -0,0 +1,14 @@
+ninja  # For faster builds.
+psutil
+ray >= 2.5.1
+pandas  # Required for Ray data.
+pyarrow  # Required for Ray data.
+sentencepiece  # Required for LLaMA tokenizer.
+numpy
+#torch == 2.1.2
+transformers >= 4.36.0  # Required for Mixtral.
+#xformers == 0.0.23.post1  # Required for CUDA 12.1.
```

Comment:
nit: May want to rephrase the comment here to mention that required functionality is integrated for HPU.

Reply:
Yes that's true. I have some changes local because I'm still testing compatibility in all possible places (tests, benchmarks)

Reply:
Can the comment thread resolutions be withheld until the changes land on the PR? The current state makes it harder to track which issues are known, since comments on their instances may be closed without a visible change.